tnt-persistence/src/main/java/org/dcsa/tnt/persistence/entity/EventCache.java

```diff
@@ -42,6 +42,9 @@ public class EventCache {
   @Column(name = "content", columnDefinition = "jsonb", nullable = false)
   private String content;
 
+  @Column(name = "document_references", columnDefinition = "text")
+  private String documentReferences;
```

Comment:
Taken line 69 from the EventCachingService do I understand correctly the list of document references is saved as key value pairs delimited with | ?

Reply:
Yes